apps/condo/domains/acquiring/constants/links.js

```diff
@@ -0,0 +1,9 @@
+// Used to navigate users to web page, where shown amount + explicit fee + confirm button
+const WEB_VIEW_PATH = '/pay/[id]'
+// Used to get explicit fee + available payment methods such as Apple Pay, web-view and etc
+const FEE_CALCULATION_PATH = '/api/get_fee/[id]'
```

Comment:
I think `/api/fee/[id]` is more RESTful as you have a clear resource `fee` and clear method `GET`

But this is not critical

Reply:
It's not giving you just fees...it also provide all existing payments methods (+urls)...Something like this
```JavaScript
{
    fee: {...},
    paymentMethods: {
        webView: { url: ... },
        applePay: { url: ... }
    },
}

```

Reply:
But you can give it more clear name 😄
Any suggestions?

Reply:
i'd say `api/pendingPayment/[id]` or `api/payment/[id] `

Reply:
changed to `api/mutli_payment/[id]`